lms/templates/video.html

```diff
@@ -152,19 +152,17 @@ <h3 class="hidden">${_('No playable video sources found.')}</h3>
 </div>
 % if cdn_eval:
 <script>
-  function sendMetricToGraphite(metricName, value) {
-    var url = "${cdn_eval_endpoint}";
-    var request = new XMLHttpRequest();
-    request.open("POST", url, true);  // send asynchronously
-    request.send(metricName + " " + value);
-  };
+  function sendPerformanceBeacon(id, expgroup, value) {
+    var data = {event: "canplaythrough", id: id, expgroup: expgroup, value: value, page: "html5vid"};
```

Comment:
did you guys agree on this event name? I guess it's not clear to me that this name means "loaded enough to be able to play"...but I admit I don't have a good alternate suggestion. "initial_video_load"?